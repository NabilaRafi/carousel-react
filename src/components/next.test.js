import React from 'react';
import renderer from 'react-test-renderer';
import Next from './next';

const suite = 'Test <Next arrow component <Next />';

describe(suite, () => {
    test('renders without breaking', () => {
        const component = renderer.create(
            <Next />
        ).toJSON();

        expect(component).toMatchSnapshot();
    })
})