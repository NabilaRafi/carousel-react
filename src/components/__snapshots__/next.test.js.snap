// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Next arrow component <Next /> renders without breaking 1`] = `
<div
  className="next-arrow"
>
  <img
    alt="next"
    src="arrow.svg"
  />
</div>
`;
